config.py

```diff
@@ -72,6 +72,13 @@
 #: How to format messages sent to Mattermost.
 MATTERMOST_TEXT_MESSAGE_FORMAT = "{emoji} **{operation}** of `{kind}` **{name}** by `{username}` in `{namespace}` {hashtag}"
 
+#: List or not used images every time Kubemen notifies about a Deployment.
+MATTERMOST_LIST_DEPLOYMENT_IMAGES = True
+
+#: Display or not a succint diff of the resource's YAML configuration when it
+#: is updated.
+MATTERMOST_DISPLAY_YAML_CONFIGURATION_DIFF = True
```

Comment:
Sounds really verbose to me, can we remove the `CONFIGURATION` part, and maybe `YAML` also?

I'd be in favor of staying consistent for attachments with something like:
* MATTERMOST_ATTACH_BADGE
* MATTERMOST_ATTACH_IMAGES
* MATTERMOST_ATTACH_DIFF